docs/samples/browser-plugin-meetings/test/wdio/spec/mediatest.js

```diff
@@ -1,41 +1,43 @@
 /* globals browser */
-import {expect} from 'chai';
+// import {expect} from 'chai';
```

Comment:
If we are not using chai's expect, is it okay to remove the commented code as well?

Reply:
done removing